.github/actions/cache-ui-dependencies/action.yaml

```diff
@@ -2,17 +2,17 @@ name: Cache UI Dependencies
 description: Cache UI Dependencies
 inputs:
   lockFile:
-    description: Where the yarn lock is written
+    description: Where the package lock is written
     required: true
-    default: "ui/apps/platform/yarn.lock"
+    default: "ui/apps/platform/package-lock.json"
 runs:
   using: composite
   steps:
     - name: Cache UI Dependencies
       uses: actions/cache@v3
       with:
         path: |
-          /github/home/.cache/yarn
+          /github/home/.npm
```

Comment:
Is `.npm` a special case that does not need subfolder of `.cache` folder?

Reply:
Yes, when running in GHA npm creates a `.npm` directory in the current $HOME.

I verified both this and the non-working `/github/home/.cache/.npm` paths by re-running the `pre-build-ui` job with debugging on and seeing exactly which files are cached.

Oddly enough, this only brings the "Fetch UI deps" part of the pipeline from ~40s to ~20s. So half the time, but not a lot of time in an absolute sense.